src/airtop/utils/process_screenshots_util.py

```diff
@@ -0,0 +1,32 @@
+from typing import List, Optional
+import base64
+from airtop import types
+
+class ProcessScreenshotsResponse:
+    def __init__(self, index: int, success: bool, binary_data: Optional[bytes] = None, error: Optional[Exception] = None):
+        self.index = index
+        self.success = success
+        self.binary_data = binary_data
+        self.error = error
+
+def process_screenshots(response: types.AiPromptResponse) -> List[ProcessScreenshotsResponse]:
+    screenshots = response.meta.screenshots
+
+    if not screenshots:
+        return []
+
+    processed_screenshots = []
+    for index, screenshot in enumerate(screenshots):
+        if not screenshot.data_url:
+            processed_screenshots.append(ProcessScreenshotsResponse(index, False, error=Exception("Screenshot data URL not found")))
+            continue
+
+        try:
+            base64_data = screenshot.data_url.replace('data:image/jpeg;base64,', '')
```

Comment:
Perhaps we could use a utility along the lines of this to try to parse the Mime type from the data_url. I know we're only returning jpegs for now but that could easily change in the future.

```
def extract_mime_and_base64(data_url: str) -> tuple[str, str]:
    """Extracts MIME type and Base64 data from a data URL, defaults to image/jpeg."""
    match = re.match(r"data:(image/\w+);base64,(.+)", data_url)
    if match:
        return match.group(1), match.group(2)
    return "image/jpeg", data_url.replace("data:image/jpeg;base64,", "")
```